core/src/epicli/cli/epicli.py

```diff
@@ -290,6 +290,8 @@ def splitarg(arg):
     sub_parser._action_groups.append(optional)
 
     def run_upgrade(args):
+        if not query_yes_no('Has backup been done?', default='no'):
```

Comment:
Maybe something more here to give some insight what/how to backup?

Reply:
@seriva Do you suggest something?

Reply:
This prompt is for all components (not only for PostgreSQL) and backup can be done in different way than using `epicli backup`, thus the prompt is so generic. For PostgreSQL, so far we don't remove anything (old data dir is preserved) so a backup may be even not required.
Does `Has backup of components to be upgraded been done?` is a better choice?